device-types/Ubiquiti/USP-RPS.yaml

```diff
@@ -0,0 +1,35 @@
+---
+manufacturer: Ubiquiti
+model: Power Backup
+slug: power-backup
```

Comment:
The slug should reflect the SKU (`USP-RPS`) or the name (`unifi-smartpower-rps`)